src/features/app/components/Form/Form.styles.ts

```diff
@@ -36,11 +44,11 @@ export const Error = styled.span`
   max-width: fit-content;
 `;
 
-export const FormContent = styled.form`
+export const FormContent = styled.form<StylesProps>`
```

Comment:
@mconiglio @Tiago-Zarzavidjian we talked about removing this prop? could this be the opportunity to do so?